kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/crypto/CryptoService.java

```diff
@@ -44,7 +46,31 @@ public interface CryptoService {
     public char[] encryptAes(char[] value) throws KuraException;
 
     /**
-     * Returns a char array based on the provided encrypted value.
+     * Returns an OutputStream that encrypts provided data using AES in the same way as the encryptAes(char[]) method,
+     * and then writes it to the supplied OutputStream lazily. *
+     * 
+     * @param streamToEncrypt
+     *            The OutputStream on which the encrypted data will be written.
+     * @return The OutputStream able to encrypt data.
+     * @throws KuraException
+     * @since 8.0
```

Comment:
Here you should put the first version of the bundle that provides the new APIs, in this case 3.0.0